newapi/ooniapi/utils.py

```diff
@@ -22,3 +23,23 @@ def nocachejson(*a, **kw):
     resp.cache_control.max_age = 0
     resp.cache_control.no_cache = True
     return resp
+
+
+def jer(error_id, tpl, *a, **kw) -> Response:
+    """Error handler with localization support
+    Return error_id, error_msg human friendly msg and errdata dict
+    Set Cache-Control: No-Cache
+    Set HTTP error code from the first optional argument or default to 400
+    """
+    # The function definition is parsed by ../scan_error_handlers.py
+    if error_id == "":
+        error_id = tpl.title().replace(" ", "")
+    if a:
+        http_code = a[0]
+    else:
+        http_code = 400
+    msg = tpl.format(kw)
+    d = dict(error=error_id, error_msg=msg, errdata=kw)
```

Comment:
IIUC here we're passing the error that occurred to the client. If my understanding is correct, then I think it's not a good idea to pass detailed error information to the consumer of a web API.